.github/workflows/release.yml

```diff
@@ -12,6 +12,15 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+        workflow_run:
+            workflows: ["Generate Manpages"]
+            types:
+              - completed
+
+      - name: Trigger Manpage Generation
+        uses: ./.github/workflows
+        with:
+          workflow: generate_manpages.yml
```

Comment:
Alsoo, if we gonna trigger the manpage generation, is the `workflow_run` here necessary? It seems to me that it will have two triggers, one after the "Generate Manpages" workflow is finished, and a second one here in this line. 

I would say that if this works out, you could just remove the `workflow_run` above and stick with this one.

Reply:
yeah looking at it now you are right I didn't even noticed that I had two triggers, I will get rid of the workflow_run code!